runestone/activecode/js/activecode.js

```diff
@@ -233,8 +232,8 @@ ActiveCode.prototype.createOutput = function () {
 
 
     var lensDiv = document.createElement("div");
-    $(lensDiv).addClass("col-md-6");
-    $(lensDiv).css("display","none");
+    $(lensDiv).addClass("col-md-6-override");
```

Comment:
Why -override?
I don't see any difference in .css("display","none") which works fine and .css({"display":"none"})  Is there a particular reason for this change?